### ISSUE TITLE
Pyodide feedback

### ISSUE THREAD

**Issue by FFY00** [FFY00/peps#2]
@hoodmane, as we discussed on PyCon US, would you be willing to review the Pyodide entry on the cross-compilation documentation PEP? :blush:

https://github.com/FFY00/peps/blob/cross-builds-info-pep/pep-0720.rst#pyodide

**Comment by hoodmane** [comment]
Thanks @FFY00. How do you want feedback on this? A first comment is you might want to add maturin to "build backends with decent cross-build support":
https://www.maturin.rs/distribution.html#cross-compiling

**Comment by FFY00** [comment]
Good catch, I had thought about it but didn't write it down and ended up forgetting :sweat_smile: 

Writing the feedback here is fine. You can also open a PR for the Pyodide section if you want to rewrite it or make any other major change, but I am happy to handle if you'd like.

**Comment by hoodmane** [comment]
cc the other Pyodide devs: @rth @ryanking13
